### ISSUE TITLE
Entwicklung von Version 1.0.0

### ISSUE THREAD

**Issue by oyooyo** [oyooyo/keyble#23]
Die deutsche Version von #22, weil die meisten keyble-Nutzer offensichtlich deutsch sprechen.
Jegliche Kommentare etc. betreffend der Entwicklung von Version 1.0.0 (siehe #22) bitte hier posten.

**Comment by frodo777** [comment]
@oyooyo gibt es schon was neues von der Version 1.0

**Comment by oyooyo** [comment]
@frodo777 Nein, leider nicht. Aktuell bin ich mehr mit anderen Dingen beschäftigt. :-(
